RELEASES.md

```
@@ -0,0 +1,66 @@
+
+
+## 2026-01-03 11:41:39
+### Changes
+Based on:
+- OpenAPI Doc  
+- Speakeasy CLI 1.680.11 (2.788.15) https://github.com/speakeasy-api/speakeasy
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)